learning_observer/learning_observer/log_event.py

```
@@ -298,3 +298,7 @@ def log_ajax(url, resp_json, request):
     )
     with open(filename, "w") as ajax_log_fp:
         ajax_log_fp.write(encoded_payload)
+
+def close_logfile(filename):
+    # remove the file from the dict storing open log files and close it
+    files.pop(filename).close()
```

There should be some check that the filename exists in our open files before we attempt to close it. Not sure if we should throw an exception if it doesn't exist or just log a warning.